release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1051,61 +1355,371 @@ The current release fixes this issue. Now, when a rollover occurs in the `indexm
 
-* Previously, the *Port* list in the *Add* flow only provided options for exposed ports and did not allow you to specify a custom port. The list has now been replaced by a typeahead select menu, and now it is possible to specify a custom port while creating the application. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1881881[*BZ#1881881*])
+* Previously, support for init containers was lost during changes to the `oc debug` command, and it was not possible to debug init containers. Support for init containers was added to the `oc debug` command, and it is now possible to debug init containers. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1909289[*BZ#1909289*])
 
-* Previously, when conditional tasks failed, the completed pipeline runs showed a permanent pending task for each failed conditional task. This issue has been fixed by disabling the failed conditional tasks and by adding skipped icons to them. This gives a better picture of the state of the pipeline run. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1880389[*BZ#1880389*])
+*OLM*
 
-* Previously, the pod scale up or down buttons were available for a single pod resource, and the page crashed when the user pressed the scale button. This issue has been fixed by not showing the scale up or down buttons for a single pod resource. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1909678[*BZ#1909678*])
+* The Marketplace Operator was written to report that the services it offered were degraded whenever the `marketplace-operator` pod exited gracefully, which would happen during routine cluster upgrades. This caused the pod to report as degraded during normal upgrades, which caused confusion. The Marketplace Operator no longer reports that it is degraded when it exits gracefully and is no longer flagged by Telemeter as degraded. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1838352[*BZ#1838352*])
 
-* Previously, the chart URL for downloading the chart to instantiate a helm release was unreachable. This happened because the `index.yaml` file from the remote repository, referenced in the Helm chart repository, was fetched and used as is. Some of these index files contained relative chart URLs. This issue has now been fixed by translating relative chart URLs to absolute URLs, which makes the chart URL reachable. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1912907[*BZ#1912907*])
+* Previously during an Operator upgrade, Operator Lifecycle Manager (OLM) deleted existing cluster service versions (CSVs) before the upgrade was completed. This caused the new CSV to be stuck in a ""Pending"" state. This bug fix updates OLM to check the ownership of the service account to ensure the new service account is created for the new CSV. As a result, existing CSVs are no longer deleted until the new CSV reaches the ""Succeeded"" state correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1857877[*BZ#1857877*])
```

Comment:
There are two sets of double quotes around Pending and Succeeded. Should only be one.